enterprise/admin.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+"""
+Django admin integration for enterprise app.
```

Comment:
@clintonb as much as I dislike non-informative docstrings myself, one of the quality checks _demand_ virtually everything to have a docstring. So if I just remove it, the build will fail. I'll try to update docstrings to carry at least some useful information, but in some cases there's no better docstring than tautology (i.e. test `setUp` methods).